rosa_tutorials/rosa-mobb-deploy-api-data-protection.adoc

```diff
@@ -0,0 +1,591 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-deploy-api-data-protection"]
+= Tutorial: Deploying OpenShift API for Data Protection on a ROSA cluster
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-deploy-api-data-protection
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-12
+//---
+//date: '2022-09-14'
+//title: Deploying OpenShift API for Data Protection on a ROSA cluster
+//tags: ["ROSA", "AWS", "STS", "OADP", "Velero", "Backup", "Restore", "Storage"]
+//authors:
+//  - Paul Czarkowski
+//  - Dustin Scott
+//---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+.Prerequisites
+
+* A ROSA classic cluster
```

Comment:
I think we can link out to the product docs until we bring this over (optional)
 
E.g., https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html